nextroute/schema/input.go

```diff
@@ -7,129 +7,217 @@ import (
 
 // Input is the default input schema for nextroute.
 type Input struct {
-	Options        any              `json:"options,omitempty"`
-	CustomData     any              `json:"custom_data,omitempty"`
-	Defaults       *Defaults        `json:"defaults,omitempty"`
-	StopGroups     *[][]string      `json:"stop_groups,omitempty"`
-	DurationMatrix *[][]float64     `json:"duration_matrix,omitempty"`
-	DistanceMatrix *[][]float64     `json:"distance_matrix,omitempty"`
+	// Options arbitrary options.
+	Options any `json:"options,omitempty"`
+	// CustomData arbitrary custom data.
+	CustomData any `json:"custom_data,omitempty"`
+	// Defaults default properties for vehicles and stops.
+	Defaults *Defaults `json:"defaults,omitempty"`
+	// StopGroups group of stops that must be part of the same route.
+	StopGroups *[][]string `json:"stop_groups,omitempty"`
+	// DurationMatrix matrix of durations in seconds between stops.
```

Comment:
I think we should also add:
"The indices of the matrix are defined as m[i][j] with i as 'from' and j as 'to'. Furthermore, the range of matrix first spans all stops of the input (same order) and then start/end locations of the vehicles in vehicle order. E.g.: [stop_1, stop_2, ..., stop_n, start_1, end_1, ..., start_m, end_m]"
Or something like this. Alternatively, we can link the docs, if we have something suitable to link to instead.